docker/Dockerfile

```diff
@@ -43,8 +43,9 @@ LABEL info.humanitarianresponse.build.date=$BUILD_DATE \
       info.humanitarianresponse.build.vcs-ref=$VCS_REF
 
 # Add PHP Soap extension to request OCHA Contribution Tracking System API.
-RUN apk add php8-soap
+RUN apk add php82-soap
```

Comment:
```
Step 18/28 : RUN apk add php82-soap
 ---> Running in 20bad5f380c4
fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/community/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz
ERROR: unable to select packages:
  php82-soap-8.2.9-r1:
    masked in: @community
    satisfies: world[php82-soap]
The command '/bin/sh -c apk add php82-soap' returned a non-zero code: 1
```
Changing to `RUN apk add php82-soap@community` works for me.

Reply:
Please check that you have the latest `public.ecr.aws/unocha/php-k8s:8.2-stable` image: `docker pull public.ecr.aws/unocha/php-k8s:8.2-stable`.

Reply:
Right, that was it, re-pulling fixed the issue.